business/proxy_status.go

```diff
@@ -48,13 +47,23 @@ func xdsStatus(sent, acked string) string {
 	return "Stale"
 }
 
-func (in *ProxyStatusService) GetConfigDump(namespace, pod string) (models.EnvoyProxyDump, error) {
-	dump, err := in.k8s.GetConfigDump(namespace, pod)
+func (in *ProxyStatusService) GetConfigDump(cluster, namespace, pod string) (models.EnvoyProxyDump, error) {
```

Comment:
This is just a question, should we use the Kiali Service account client here? 
Also, should this work in a primary remote deployment model? As the remote wouldn't have an Istio